neo/Helper.cs

```diff
@@ -98,6 +98,16 @@ internal static string GetVersion(this Assembly assembly)
             return (string)attribute.ConstructorArguments[0].Value;
         }
 
+        public static string ToBase64String(this byte[] value)
```

Comment:
We don't need this function, I think. Just use `Convert` directly.

Reply:
I added this because the argument, in order to prevent mistakes

Reply:
I think the default value is `Base64FormattingOptions.None`.